src/__tests__/helpers/expectedFixtures.ts

```diff
@@ -0,0 +1,302 @@
+export const expectedMultipleEvents = {
```

Comment:
```suggestion
export const expectedMultipleEventsResponse = {
```
and rename the file to `expectedAssetServiceResponses`